src/main/java/com/conveyal/gtfs/graphql/GraphQLGtfsSchema.java

```diff
@@ -644,6 +651,20 @@
                     // DataFetchers can either be class instances implementing the interface, or a static function reference
                     .dataFetcher(new JDBCFetcher("patterns"))
                     .build())
+            .field(newFieldDefinition()
+                .name("shapes")
```

Comment:
To avoid confusion with existing schema items with the `shape(s)` name (such as `tripType#shape`), perhaps it would be good to rename this `shape_encoded_polyline` or something.